packages/flutter/test/material/slider_test.dart

```diff
@@ -607,7 +607,7 @@ void main() {
                 data: theme,
                 child: Slider(
                   value: value,
-                  label: '$value',
+                  label: value,
```

Comment:
Value is a double.

The label has to be a string.

Reply:
@dnfield Thank you for your quick review.
Sorry, I missed the check.